src/lib.rs

```diff
@@ -1065,23 +1065,7 @@ impl Client {
 
     ///
     /// # Errors
-    pub async fn get_remote_wasm(&self, contract_id: &[u8; 32]) -> Result<Vec<u8>, Error> {
```

Comment:
We can't remove `get_remote_wasm` in a minor version. The next major version will be timed with the v22 protocol release. So all we can do with this function right now after copying it to the CLI is marking it as deprecated here:

```suggestion
    #[deprecated(note = "use get_ledger_entries()")]
    pub async fn get_remote_wasm(&self, contract_id: &[u8; 32]) -> Result<Vec<u8>, Error> {
```

Reply:
Makes sense. I though this lib was only for internal use, but if we expose it externally then yeah we can't remove it

Reply:
The lib is published on crates.io and docs for it are publicly available on docs.rs.